optimum/exporters/onnx/model_configs.py

```diff
@@ -723,6 +723,27 @@ def generate_dummy_inputs(self, framework: str = "pt", **kwargs):
         return dummy_inputs
 
 
+class CLIPVisionOnnxConfig(VisionOnnxConfig):
+    ATOL_FOR_VALIDATION = 1e-3
+    # The ONNX export of this architecture needs the Trilu operator support, available since opset 14
+    DEFAULT_ONNX_OPSET = 14
+
+    NORMALIZED_CONFIG_CLASS = NormalizedVisionConfig
+
+    @property
+    def inputs(self) -> Dict[str, Dict[int, str]]:
+        return {
+            "pixel_values": {0: "image_batch_size", 1: "num_channels", 2: "height", 3: "width"},
+        }
+
+    @property
+    def outputs(self) -> Dict[str, Dict[int, str]]:
+        return {
+            "last_hidden_state": {0: "batch_size", 1: "sequence_length"},
+            "pooler_output": {0: "batch_size"},
+        }
```

Comment:
```suggestion
class CLIPVisionOnnxConfig(ViTOnnxConfig):
    pass
```
Just inheriting from ViT's ONNX config seems to work according to the discussion in this issue: https://github.com/huggingface/optimum/issues/1186#issuecomment-1637641760